mcp_ai_agents/sandbase_cli_mcp_agent/README.md

```
@@ -0,0 +1,52 @@
+python main.py
+```
+
+The script starts the immutable SandBase CLI release asset with `npx`, asks the agent to list compatible models, and sends one short prompt through the selected model. Review the tool calls and do not use production data for the first run.
+
+## How it works
+
+`main.py` uses `MCPServerStdio` to launch `npx -y https://github.com/sandbaseai/cli/releases/download/v0.1.17/sandbaseai-cli-0.1.17.tgz connect`. SandBase handles provider/model discovery and OAuth; the OpenAI Agents SDK handles the example agent loop. No SandBase provider key is committed or placed in the project files.
+
+## Project structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)